pkg/reconciler/kubernetes/tektondashboard/controller.go

```diff
@@ -64,13 +65,14 @@ func NewExtendedController(generator common.ExtensionGenerator) injection.Contro
 			logger.Fatal(err)
 		}
 
+		tisClient := operatorclient.Get(ctx).OperatorV1alpha1().TektonInstallerSets()
 		c := &Reconciler{
-			kubeClientSet:      kubeClient,
+			pipelineInformer:   tektonPipelineInformer,
+			installerSetClient: client.NewInstallerSetClient(tisClient, operatorVer, dashboardVer, v1alpha1.KindTektonDashboard, nil),
```

Comment:
for metrics instead of passing nil
use https://github.com/tektoncd/operator/blob/main/pkg/reconciler/openshift/openshiftpipelinesascode/metrics.go
move the that to a common directory
and use that as `common.NoMetrics`